src/torrent.cpp

```diff
@@ -651,8 +647,9 @@ bool is_downloading_state(int const st)
 		if (m_trackers.empty()) return true;
 		if (!settings().get_bool(settings_pack::use_dht_as_fallback)) return true;
 
-		return std::none_of(m_trackers.begin(), m_trackers.end()
-			, [](aux::announce_entry const& tr) { return bool(tr.verified); });
+		using std::begin;
```

Comment:
Is it needed?

Reply:
ah, no. those are left-overs from an earlier implemetnation